src/libraries/System.Text.Json/src/System/Text/Json/Reader/JsonReaderException.cs

```diff
@@ -10,7 +10,7 @@ namespace System.Text.Json
     [Serializable]
     internal sealed class JsonReaderException : JsonException
     {
-        public JsonReaderException(string message, long lineNumber, long bytePositionInLine) : base(message, path: null, lineNumber, bytePositionInLine)
+        public JsonReaderException(string message, long lineNumber, long bytePositionInLine) : base(message, path: string.Empty, lineNumber, bytePositionInLine)
```

Comment:
This change seems unnecessary to me.

Reply:
yes, same as above, it was nullable then changed to this, if we want to change it back to nullable let me know